cdpctl/command/validate.py

```diff
@@ -50,11 +50,18 @@
 import cdpctl.validation as validation
 from cdpctl import SUPPORTED_PLATFORMS
 from cdpctl.utils import load_config
-from cdpctl.validation import UnrecoverableValidationError, conftest
+from cdpctl.validation import UnrecoverableValidationError, conftest, get_issues
 from cdpctl.validation.aws_utils import validate_aws_config
+from cdpctl.validation.renderer import get_renderer
 
 
-def run_validation(target: str, config_file: str, debug: bool = False) -> None:
+def run_validation(
+    target: str,
+    config_file: str,
+    debug: bool = False,
+    output_format: str = "text",
+    output_file: str = "-",
```

Comment:
why not blank as default?

Reply:
because convention for Unix usually is - meaning stdout.  for tar etc.